src/dropzone/DropzoneDirective.ts

```diff
@@ -47,7 +47,10 @@ export class DropzoneDirective implements ng.IDirective {
       'sending': (file, xhr, formData) => {
         scope.$apply(() => {
           // set form data prior to submitting the dz form
-          scope.$ctrl.backend.updateFormData(scope.$ctrl.folder, file, formData);
+          scope.$ctrl.backend.updateFormData(scope.$ctrl.folder, file, formData)
```

Comment:
Should we be handling the situation where the promise rejects?  Or is this not needed since we always resolve the promise.

Reply:
That's right, the 'updateFormData' always resolves and there is never a reject case to handle. Also, same as the previous comment, since we aren't using dropzonejs to upload files, the 'sending' event does not get called. Added a commit removing the 'sending' event handler.